node/src/components/consensus/error.rs

```diff
@@ -0,0 +1,38 @@
+use std::collections::BTreeMap;
+
+use num::rational::Ratio;
+use thiserror::Error;
+
+use crate::types::BlockSignatures;
+use casper_types::{PublicKey, U512};
+
+#[derive(Error, Debug)]
+pub enum FinalitySignatureError {
+    #[error(
+        "Block signatures contain bogus validator. \
+         trusted validator weights: {trusted_validator_weights:?}, \
+         block signatures: {block_signatures:?}, \
+         bogus validator public key: {bogus_validator_public_key:?}"
+    )]
+    BogusValidator {
+        trusted_validator_weights: BTreeMap<PublicKey, U512>,
+        block_signatures: Box<BlockSignatures>,
```

Comment:
I don't think we need all this information in the error.

Reply:
It's been helpful to me...

Reply:
I'm for having it here - if we get that error in production, we better know exactly what signature was the problem, who signed it and whether they truly weren't a validator.

Reply:
ok